README.md

```diff
@@ -84,6 +84,7 @@ module.exports = function(defaults) {
     'ember-app-shell': {
       visitPath: '/my-app-shell',
       outputFile: 'my-app-shell.html',
+      disableSandbox: false,
```

Comment:
wouldn't it be best to have a more generic `chromeFlags` array that can be merged with the default flags?

Reply:
That's the first approach I took.. I switched to this because I thought this would give the user too much power. If you think it would be OK I can change it of course.

Reply:
I'll leave it to @martndemus to make the call

Reply:
I'm +1 on just specifying generic `chromeFlags`.

Reply:
Would you define `--headless` and `--disable-gpu` as default value for this `chromeFlags` option which could be overwritten or add those always?

Reply:
I'm thinking add always.